ee/tabby-schema/src/schema/mod.rs

```diff
@@ -541,6 +541,25 @@ impl Query {
             .await
     }
 
+    async fn resolve_git_url(ctx: &Context, git_url: String) -> Result<bool> {
+        let user = check_user_and_auth_token(ctx).await?;
+
+        let repositorys = ctx
```

Comment:
use https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/3df11d1eb3c3d652ef715f644eb1f44730ff255f/crates/tabby-common/src/axum.rs#L58 to find the closest matched git url